Source/Core/InputCommon/ControllerInterface/GameController/GameController.mm

```diff
@@ -0,0 +1,92 @@
+// Copyright 2013 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#import <objc/message.h>
+
+#include "InputCommon/ControllerInterface/GameController/GameController.h"
+#include "InputCommon/ControllerInterface/GameController/GameControllerDevice.h"
+
+#include "Common/Logging/Log.h"
+
+namespace ciface::GameController
+{
+static void AddController(GCController* controller)
+{
+  if (!@available(macOS 11.0, *)) {
```

Comment:
You can't negate an ``@available`` check. An alternative might be
```objc
if (@available(macOS 11.0, *))
{
}
else
{
    INFO_LOG_FMT(...);
    return;
}
```

But admittedly, it is a bit ugly.